integration-tests/tests/snap_op_test.go

```diff
@@ -111,7 +111,7 @@ func (s *snapOpSuite) TestRemoveBusyRetries(c *check.C) {
 		output := cli.ExecCommand(c, "snap", "changes")
 		id := regexp.MustCompile(`(?m)([0-9]+).*Doing.*Remove.*"`).FindStringSubmatch(output)[1]
 		needle = `will retry: `
-		wait.ForCommand(c, needle, "snap", "changes", id)
+		wait.ForCommand(c, needle, "snap", "change", id)
```

Comment:
I wonder if it would make sense to add an integration tests for `snap changes hello-world` ? But easy enough to do in a followup branch.